app/assets/stylesheets/wipe.css.scss

```diff
@@ -0,0 +1,237 @@
+  left     : -77px;
+  z-index  : 10;
+  width    : 100px;
+}
+#overlay-down-arrow{
+  position            : absolute;
+  bottom              : 60px;
+  left                : 50%;
+  height              : 20px;
+  width               : 40px;
+  margin              : auto;
+  margin-left         : -20px;
+  cursor              : pointer;
+  background-image    : url("down-arrow.svg");
+  background-position : center;
+  background-repeat   : no-repeat;
+  color               : $theme-red-blue-3;
+  text-align          : center;
+  -webkit-animation   : bounce 2s infinite;
+  -moz-animation      : bounce 2s infinite;
+  animation           : bounce 2s infinite;
+}
+   /* Mobile */
+@media (max-width: $screen-xs-max) {
+  #overlay-description{
+    padding   : 10px;
+    font-size : 14px;
+  }
+  #overlay-title{
+    padding   : 10px;
+    font-size : 26px;
+  }
+  #testimonials {
+    position       : static;
+    padding-top    : 12px;
+    cursor         : pointer;
+    font-size      : 12px;
+    color          : #fff;
+    text-align     : center;
+    text-transform : uppercase;
+  }
+  #testimonials:after{
+    opacity : 0;
+  }
+  #testimonial-span{
+    padding : 5px;
+  }
+  #overlay-down-arrow{
+   /* margin-top : 20px;*/
+  }
+  #overlay-arrow-text {
+   /* margin-top : 40px;*/
+  }
+  #homesplash{
+    background              : #efefee image-url('nyc_snow-edited_crop_small.jpg') no-repeat top center;
+    background-attachment   : fixed;
+    background-size         : cover;
+    -webkit-background-size : cover;
+    -o-background-size      : cover;
+    -moz-background-size    : cover;
+  }
+}
+   /* Float Shadow */
+.hvr-float-shadow {
+  position                    : relative;
+  display                     : inline-block;
+  vertical-align              : middle;
+  box-shadow                  : 0 0 1px rgba(0, 0, 0, 0);
+  -webkit-transition-property : transform;
+  transition-property         : transform;
+  -webkit-transition-duration : 0.3s;
+  transition-duration         : 0.3s;
+  -webkit-transform           : translateZ(0);
+  transform                   : translateZ(0);
+  -webkit-backface-visibility : hidden;
+  backface-visibility         : hidden;
+  -moz-osx-font-smoothing     : grayscale;
+}
+.hvr-float-shadow:before {
+  content                     : '';
+  position                    : absolute;
+  top                         : 100%;
+  left                        : 5%;
+  z-index                     : -1;
+  height                      : 10px;
+  width                       : 90%;
+  opacity                     : 0;
+  background                  : -webkit-radial-gradient(center, ellipse, rgba(0, 0, 0, 0.35) 0%, rgba(0, 0, 0, 0) 80%);
+  background                  : radial-gradient(ellipse at center, rgba(0, 0, 0, 0.35) 0%, rgba(0, 0, 0, 0) 80%);
+  -webkit-transition-property : transform, opacity;
+  transition-property         : transform, opacity;
+  /* W3C */
+  -webkit-transition-duration : 0.3s;
```

Comment:
Property name should be immediately followed by a colon<br>Avoid vendor prefixes.